data/tokyo/aws-loft-tokyo.md

```diff
@@ -61,4 +65,4 @@ Easily one of the best places to work from in Tokyo. It's free to use, all you n
 
 **Meeting rooms**: There are several meeting rooms that (I think) can be booked for free as well. I haven't really looked into it yet, though.
 
-**Overall**: A really cool space, comfortable work area, good location right by Meguro station. The main drawback is that the space is only open from 10-18 weekdays (closed on weekends). Definitely a good alternative to Lodge, which has become very crowded lately.
\ No newline at end of file
+**Overall**: A really cool space, comfortable work area, good location right by Meguro station. The main drawback is that the space is only open from 10:00-18:00 weekdays, and is closed on weekends. Definitely a good alternative to Lodge, which has become very crowded lately.
```

Comment:
I heard Lodge was closed, but I haven't verified. Might be nice to create hyperlinks between these files/links here, but the client front-end would probably need to translate these into a different link format.

Reply:
Yeah, I've already flagged Lodge as permanently closed. I should update the text to remove the reference. Markdown links are supported.